eagleEvents/routes/events.py

```diff
@@ -17,43 +21,109 @@ def list_events():
     return render_template('event.html.j2')
 
 
-def allowed_file(filename):
-    return '.' in filename and filename.rsplit('.', 1)[1].lower() in set(['csv'])
-
-
-@events_blueprint.route('/modifyEvent', methods=['GET', 'POST'])
+@events_blueprint.route('/addEvent', methods=['GET', 'POST'])
+@multi_auth.login_required
+def add_event():
+    planner_list = User.query.all()
+    customer_list = Customer.query.all()
+    sizes = TableSize.query.all()
+    # create a new event on the first customer in the list
+    event = Event(customer_list[0])
+    # set the default time to 7 days from now at 7 PM
+    time = datetime.datetime.now()
+    time = time.replace(time.year, time.month, time.day+7, hour=19, minute=0)
+    event.time = time
+    # if the method is post, handle that the same way as modify event
+    if request.method == 'POST':
+        # let the function tell us how we should be directed
+        handler = handle_post(event, True)
+        # handle the redirection
+        return handler
+    # if this is GET, handle that for first time event creation
+    return render_template('add-update-event.html.j2', event=event, sizes=sizes, planner=g.current_user,
+                           customers=customer_list, planners=planner_list, new=True, imported=False,
+                           date=convert_time(event.time), cancel_redirect=url_for('events.list_events'))
+
+
+@events_blueprint.route('/modifyEvent/<event_id>', methods=['GET', 'POST'])
 @multi_auth.login_required
-def modify_event():
-    # TODO get the event that is currenlty being modified
-    event = Event.query.first()
-    if request.method == 'POST' and request.files:
-        # check if the post request has the file part
-        if 'file' not in request.files:
-            flash('No file part')
-            return redirect(request.url)
-        file = request.files['file']
-        # if user does not select file, browser also
-        # submit a empty part without filename
-        if file.filename == '':
-            flash('No selected file')
-            return redirect(request.url)
-        if file and allowed_file(file.filename):
-            filename = secure_filename(file.filename)
-            home = str(Path.home())
-            path = os.path.join(home, filename)
-            file.save(path)
-            c = g.current_user.company
-            c.process_guest_list(path, event)
-            flash('Import Completed: ' + filename)
+def modify_event(event_id):
+    event = Event.query.get(event_id)
+    planner_list = User.query.all()
+    customer_list = Customer.query.all()
```

Comment:
Need to filter by current user's company due to the possibility of having multiple companies with users in the system

Reply:
I feel like you told me that Tuesday and I just completely ignored you. 👍 
I've changed it